dotCMS/src/main/java/com/dotmarketing/portlets/osgi/AJAX/OSGIAJAX.java

```diff
@@ -266,4 +267,19 @@ private void remove () {
 
     }
 
+    @Override
+    protected Set<String> getAllowedCommands() {
+        return Set.of(
```

Comment:
shouldn't be a constant static

Reply:
I had it as a class field. non-static but Mockito  wouldnt see the results when calling the real method so ended up inlining the Set instantiation. It isn't a big penalti performance wise